setup.py

```diff
@@ -178,7 +178,7 @@ def run(self):
       classifiers=classifiers,
       cmdclass=cmdclass,
       # When changing this, also update test-requirements.txt.
-      install_requires=['typed-ast >= 1.3.5, < 1.4.0; python_version < "3.8"',
+      install_requires=['typed-ast >= 1.3.5, < 1.4.0',
```

Comment:
I wonder if that does not make sense though in general?
Even when assuming that it gets used with `--python-version` / for versions before py38, it could still be an optional dependency then, being checked during runtime?!